cmd/server/main.go

```diff
@@ -35,6 +37,7 @@ var (
 	psqlDsn             = flag.String("psql", os.Getenv("POSTGRES_DSN"), "Postgres DSN")
 	debugPtr            = flag.Bool("debug", defaultDebug, "print debug output")
 	logJSONPtr          = flag.Bool("log-json", defaultLogJSON, "log in JSON")
+	ServiceName         = flag.String("Service-Name", defaultServiceName, " ServiceName/deployment name")
```

Comment:
flag should be lowercase. also please improve the help text, it's not clear from that alone what this flag does